tests/e2e/utils/src/flows/constants.js

```diff
@@ -2,32 +2,60 @@
-export const WP_ADMIN_ANALYTICS_PAGES = baseUrl + 'wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2F';
-export const WP_ADMIN_ALL_USERS_VIEW = baseUrl + 'wp-admin/users.php';
-export const WP_ADMIN_IMPORT_PRODUCTS = baseUrl + 'wp-admin/edit.php?post_type=product&page=product_importer';
-
+export const WP_ADMIN_DASHBOARD = baseUrl + 'wp-admin/';
+export const WP_ADMIN_PLUGINS = WP_ADMIN_DASHBOARD + 'plugins.php';
+export const WP_ADMIN_PERMALINK_SETTINGS = WP_ADMIN_DASHBOARD + 'options-permalink.php';
+export const WP_ADMIN_ALL_USERS_VIEW = WP_ADMIN_DASHBOARD + 'users.php';
+/**
+ * WooCommerce core post type pages.
+ * @type {string}
+ */
+export const WP_ADMIN_POST_TYPE = WP_ADMIN_DASHBOARD + 'edit.php?post_type=';
+export const WP_ADMIN_NEW_POST_TYPE = WP_ADMIN_DASHBOARD + 'post-new.php?post_type=';
+export const WP_ADMIN_ALL_COUPONS_VIEW = WP_ADMIN_POST_TYPE + 'shop_coupon';
+export const WP_ADMIN_NEW_COUPON = WP_ADMIN_NEW_POST_TYPE + 'shop_coupon';
+export const WP_ADMIN_ALL_ORDERS_VIEW = WP_ADMIN_POST_TYPE + 'shop_order';
+export const WP_ADMIN_NEW_ORDER = WP_ADMIN_NEW_POST_TYPE + 'shop_order';
+export const WP_ADMIN_ALL_PRODUCTS_VIEW = WP_ADMIN_POST_TYPE + 'product';
+export const WP_ADMIN_NEW_PRODUCT = WP_ADMIN_NEW_POST_TYPE + 'product';
+export const WP_ADMIN_IMPORT_PRODUCTS = WP_ADMIN_ALL_PRODUCTS_VIEW + '&page=product_importer';
+/**
+ * WooCommerce settings pages.
+ * @type {string}
+ */
+export const WP_ADMIN_PLUGIN_PAGE = WP_ADMIN_DASHBOARD + 'admin.php?page=';
+export const WP_ADMIN_WC_HOME = WP_ADMIN_PLUGIN_PAGE + 'wc-admin';
+export const WP_ADMIN_SETUP_WIZARD = WP_ADMIN_WC_HOME + '&path=%2Fsetup-wizard';
+export const WP_ADMIN_ANALYTICS_PAGES = WP_ADMIN_WC_HOME + '&path=%2Fanalytics%2F';
+export const WP_ADMIN_WC_SETTINGS = WP_ADMIN_PLUGIN_PAGE + 'wc-settings&tab=';
+export const WP_ADMIN_NEW_SHIPPING_ZONE = WP_ADMIN_WC_SETTINGS + 'shipping&zone_id=new';
+/**
+ * Shop pages.
+ * @type {string}
+ */
 export const SHOP_PAGE = baseUrl + 'shop';
 export const SHOP_PRODUCT_PAGE = baseUrl + '?p=';
 export const SHOP_CART_PAGE = baseUrl + 'cart';
 export const SHOP_CHECKOUT_PAGE = baseUrl + 'checkout/';
 export const SHOP_MY_ACCOUNT_PAGE = baseUrl + 'my-account/';
+/**
+ * Customer account pages.
+ * @type {string}
+ */
+export const MY_ACCOUNT_ORDERS = SHOP_MY_ACCOUNT_PAGE + 'orders';
+export const MY_ACCOUNT_DOWNLOADS = SHOP_MY_ACCOUNT_PAGE + 'downloads';
+export const MY_ACCOUNT_ADDRESSES = SHOP_MY_ACCOUNT_PAGE + 'edit-address';
+export const MY_ACCOUNT_ACCOUNT_DETAILS = SHOP_MY_ACCOUNT_PAGE + 'edit-account';
 
-export const MY_ACCOUNT_ORDERS = baseUrl + 'my-account/orders';
-export const MY_ACCOUNT_DOWNLOADS = baseUrl + 'my-account/downloads';
-export const MY_ACCOUNT_ADDRESSES = baseUrl + 'my-account/edit-address';
-export const MY_ACCOUNT_ACCOUNT_DETAILS = baseUrl + 'my-account/edit-account';
+/**
+ * Test control flags.
```

Comment:
Do you think in the future we'll want to include the `SMOKE_TEST_*` variables in here?

Reply:
> Do you think in the future we'll want to include the SMOKE_TEST_* variables in here?

The smoke test variables all replace counterparts in `default.json`. I'm not sure there is a need to be able to determine whether the config value is the env variable or from the config file.